proxy/src/transport/tls/config.rs

```diff
@@ -271,36 +269,40 @@ impl CommonConfig {
 
 }
 
+pub fn watch_for_config_changes(settings: Option<&CommonSettings>)
```

Comment:
Nit: maybe this ought to get a doc comment, since all the other (now private) functions in the module do?

Reply:
I honestly couldn't think of a comment that would be useful beyond the naming and type signature.

Reply:
Reasonable.